plugins/search/src/alpha.tsx

```diff
@@ -154,6 +155,11 @@ export const SearchPage = createPageExtension({
                         name: 'Architecture Decision Records',
                         icon: <DocsIcon />,
                       },
+                      {
+                        value: 'stack-overflow',
+                        name: 'Stack Overflow',
+                        icon: <StackOverflowIcon />,
```

Comment:
Should the icon be extracted to a react package to avoid direct dependency between plugins?

Reply:
Removed, once we are not going to add stack overflow in the default search page 🙂 : https://github.com/backstage/backstage/pull/20456#discussion_r1350405111